backend/app.hopps.org/src/main/java/app/hopps/org/rest/OrganizationResource.java

```diff
@@ -63,6 +69,29 @@ public Response getOrganizationBySlug(@PathParam("slug") String slug) {
         return Response.ok(organization).build();
     }
 
+    @GET
+    @Path("/my")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get my organization", description = "Retrieves the details of an organization the current user is assigned to.")
+    @APIResponse(responseCode = "200", description = "Own organization retrieved successfully", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = Organization.class)))
+    @APIResponse(responseCode = "404", description = "Organization not found for user")
+    public Organization getMyOrganization(@Context SecurityContext securityContext) {
```

Comment:
Könnte man das nicht auf der Klasse auch injecten?

Reply:
Da wir es nur in diesem Endpunkt brauchen, macht es kein großen Sinn dies in der Klasse zu injecten, wenn wir dies mehrmals brauchen macht es mehr sinn als aktuell.